### PR TITLE
Adding Gitlab and Azure to CI/CD options

### LINKED ISSUE #80: Gitlab CI support
Currently the CI/CD can only generate Travis/Jenkins Configuration. It would be cool to have it also generate gitlab ci configuration. What do you think?

### PR DESCRIPTION
**Pull request by angeliski**
Refactoring the CI/CD options to a better experience in add another tool.

This resolve #80

@pascalgrimaud You can review the Azure description? I wrote but I don't know it's correct

### CODE REVIEW

File: src/main/webapp/app/home/ci-cd/ci-cd.component.html [hunk 1]
```diff
@@ -22,14 +22,29 @@ <h3>Travis</h3>
...
                 </div>
+                <div class="col-md-6">
+                    <h3><a href="https://gitlab.com/">Gitlab</a></h3>
+                    <ul>
+                        <li>A <code>.gitlab-ci.yml</code> will be added to your project.</li>
+                        <li>Your PR now is running using the GitLab Continuous Integration</li>
+                    </ul>
+                </div>
+                <div class="col-md-6">
+                    <h3><a href="https://azure.microsoft.com/">Azure</a></h3>
```
Comment (pascalgrimaud) [inline src/main/webapp/app/home/ci-cd/ci-cd.component.html]: the URL should be: https://dev.azure.com with text: Azure Pipelines

File: src/main/webapp/app/home/ci-cd/ci-cd.component.html [hunk 2]
```diff
@@ -22,14 +22,29 @@ <h3>Travis</h3>
...
+                    <ul>
+                        <li>A <code>azure-pipelines.yml</code> will be added to your project.</li>
+                        <li>Go to your Azure instance</li>
```
Comment (pascalgrimaud) [inline src/main/webapp/app/home/ci-cd/ci-cd.component.html]: Go to your project in your Azure DevOps organization.

File: src/main/webapp/app/home/ci-cd/ci-cd.component.html [hunk 3]
```diff
@@ -44,6 +59,8 @@ <h3>Jenkins</h3>
                                 #ctrl="ngModel">
                             <option value="travis">Travis</option>
                             <option value="jenkins">Jenkins</option>
+                            <option value="gitlab">Gitlab</option>
+                            <option value="azure">Azure</option>
```
Comment (pascalgrimaud) [inline src/main/webapp/app/home/ci-cd/ci-cd.component.html]: Azure Pipelines

**Comment by angeliski** [comment]
Done @pascalgrimaud !
Thanks for your review

**Comment by jdubois** [comment]
Awesome! I'm merging & testing, will do a release if everything is fine.
Thanks a lot @angeliski !!

**Pull request merged**